src/main/resources/db/changeLog.xml

```diff
@@ -931,25 +931,93 @@
             <column name="defaultresources" type="longtext"/>
         </addColumn>
     </changeSet>
-
-    <changeSet id="remove_group_contentversion" author="katja danilova">
+    
+<changeSet id="remove_group_contentversion" author="katja danilova">
     <addColumn tableName="contentversion">
-            <column name="status" type="varchar(191)"/>
-            <column name="devicegroup_id" type="binary(16)">
-                <constraints foreignKeyName="FK_CONTENT_VERSION_DEVICE_GROUP_ID" referencedColumnNames="id" referencedTableName="exhibitiondevicegroup"/>
-            </column>
-        </addColumn>
+        <column name="status" type="varchar(191)" />
+        <column name="devicegroup_id" type="binary(16)" />
+    </addColumn>
+
+    <addForeignKeyConstraint 
+        baseTableName="contentversion" 
+        baseColumnNames="devicegroup_id" 
+        constraintName="FK_CONTENT_VERSION_DEVICE_GROUP_ID" 
+        referencedTableName="exhibitiondevicegroup" 
+        referencedColumnNames="id" />
+
+    <addColumn tableName="groupcontentversion">
+        <column name="language" type="varchar(191)" />
+    </addColumn>
+    <addColumn tableName="groupcontentversion">
+        <column name="activeConditionUserVariable" type="varchar(191)" />
+    </addColumn>
+    <addColumn tableName="groupcontentversion">
+        <column name="activeConditionEquals" type="varchar(191)" />
+    </addColumn>
+
+    <dropForeignKeyConstraint 
+        baseTableName="groupcontentversion" 
+        constraintName="FK_GROUP_CONTENT_VERSION_CONTENT_VERSION_ID" />
+    <dropForeignKeyConstraint 
+        baseTableName="contentversionroom" 
+        constraintName="FK_CONTENT_VERSION_ROOM_CONTENTVERSION_ID" />
+    <dropForeignKeyConstraint 
+        baseTableName="exhibitionpage" 
+        constraintName="FK_EXHIBITION_PAGE_CONTENT_VERSION_ID" />
+
+    <sql>
+        -- Clean up data with SQL commands
```

Comment:
This comment is a bit vague. Maybe something more like "Delete content versions without group content version"